perses/tests/test_cli.py

```diff
@@ -0,0 +1,47 @@
+from click.testing import CliRunner
+from perses.app.cli import cli
+
+test_yaml = """
+protein_pdb: Tyk2_protein.pdb
+ligand_file: Tyk2_ligands_shifted.sdf
+old_ligand_index: 0
+new_ligand_index: 3
+forcefield_files:
+  - amber/ff14SB.xml
+  - amber/tip3p_standard.xml
+  - amber/tip3p_HFE_multivalent.xml
+  - amber/phosaa10.xml
+small_molecule_forcefield: openff-1.0.0
+pressure: 1
+temperature: 300
+solvent_padding: 9
+atom_expression:
+  - IntType
+bond_expession:
+  - DefaultBonds
+n_steps_per_move_application: 1
+fe_type: repex
+checkpoint_interval: 50
+n_cycles: 1
+n_states: 3
+n_equilibration_iterations: 0
+trajectory_directory: temp/offlig10to24
+trajectory_prefix: out
+atom_selection: not water
+phases:
+  - complex
+  - solvent
+  - vacuum
+timestep: 4
+h_constraints: true
+"""
```

Comment:
From what I can see, this is the same yaml file in the `examples/protein-ligand/cli/protein-ligand.yaml` path. I wonder if we want to use that file directly to avoid syncronization and redundency issues once things change. So, something like `python cli.py --yaml-path=/path/to/perses/examples/protein-ligand/cli/protein-ligand.yaml`

Reply:
Or whatever fix the test in this case, but I guess the point still stands on using the file we already have instead of copying it.

Reply:
@ijpulidos since our examples are outside our package source, I'm not sure the most reliable way to get the location of that yaml, do you have any ideas? Normally I would use something like this https://docs.python.org/3.8/library/importlib.html#importlib.resources.path but I would like to avoid doing something like `pathlib.Path(__file__).joinpath("../../../").resolve()` if I can help it.

Reply:
Or https://setuptools.pypa.io/en/latest/userguide/datafiles.html

Reply:
Actually never mind to trying to get this into the 0.9.4 release, if it ends up in that release that is fine, but once I get this PR merged I can just make a docker image with the `dev` label and share that

Reply:
But I still like the idea of making an issue to fix all the tests, and just keep the yaml hard coded for now

Reply:
Yes, I agree! I created the issue regarding this. CC #942